index.js

```diff
@@ -70,7 +72,7 @@ connection`),
     is: true,
     then: Joi.string().required(),
   }).description('Used when useIAM is true. The AWS region name.'),
-  ssl: Joi.string().description(`Typically 'Amazon RDS'`),
+  ssl: Joi.alternatives(Joi.string(), Joi.object()).description(`Typically 'Amazon RDS'`),
```

Comment:
Please change description to "Either Amazon RDS or an object containing ssl-related fields"